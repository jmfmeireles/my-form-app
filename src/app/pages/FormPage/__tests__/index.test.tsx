import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { FormPage } from '..';

const shallowRenderer = createRenderer();

describe('<FormPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<FormPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
