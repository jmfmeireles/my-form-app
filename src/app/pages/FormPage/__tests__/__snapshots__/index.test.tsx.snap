// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Form Page
    </title>
    <meta
      content="A page where you can insert and check data from people"
      name="description"
    />
  </Helmet>
  <Header />
  <styles__PageWrapper>
    <Form />
    <styles__TableWrapper>
      <DataTable
        isFormPage={true}
      />
    </styles__TableWrapper>
    <ToastMessage />
  </styles__PageWrapper>
  <Footer />
</React.Fragment>
`;
